### ISSUE TITLE
When "Unsuppressed" and "Suppressed" control receives the keyboard focus, the color contrast ratio of border with respect to background is less than 3:1.

### ISSUE THREAD

**Issue by shreyashi227** [microsoft/sarif-web-component#26]
### Actual:
When "Unsuppressed" and "Suppressed" control receives the keyboard focus, the color contrast ratio of border with respect to background is 1.627:1 which is less than 3:1.

### Expected:
When "Unsuppressed" and "Suppressed" control receives the keyboard focus, the color contrast ratio of border with respect to background should be greater than or equal to 3:1.

**Comment by jeffersonking** [comment]
I see a dark blue border with contrast ratio of 4.53:1. 

![Screen Shot 2020-12-30 at 12 51 12 AM](https://user-images.githubusercontent.com/3452831/103340769-c1350200-4a39-11eb-9d6d-e4df782c9a67.png)

Note: I am not referring to the decorative light blue "glow".

**Comment by shreyashi227** [comment]
@jeffersonking 
When we expand this "Suppression" control using enter two list items appears and when we move the keyboard focus on those two list items "Unsuppressed" and "Suppressed" then the color contrast ratio of keyboard focus border on "Unsuppressed" and "Suppressed" control with respect to background is less than 3:1.

![image](https://user-images.githubusercontent.com/60881447/103348031-c7b98d00-4abe-11eb-8d73-7829e7ea28e3.png)

**Comment by jeffersonking** [comment]
Fixed in v0.5 (2413b17b8cf2f5a11cf4238df5418f00eddd43a1). Thanks for reporting!
